docs/historical/performance.rst

```diff
@@ -4,82 +4,70 @@ Historical: Performance and Scalability
 =======================================
 
 .. note::
-   This scalability review summarises results in a paper, Parsl: Pervasive
-   Parallel Programming in Python, which was published in 2019. The results
-   have not been updated since then. For that reason, this section is marked
-   as historical.
+   This scalability review summarises results in a paper, Parsl: Pervasive Parallel Programming in
+  Python, which was published in 2019. The results have not been updated since then. For that reason,
```

Comment:
```
 Warning, treated as error:
/home/runner/work/parsl/parsl/docs/historical/performance.rst:8:Block quote ends without a blank line; unexpected unindent.
```

I think this line and the subsequent one should all be indented to the same depth as line 7